checker/tests/nullness/Issue1142.java

```diff
@@ -0,0 +1,14 @@
+// Issue 1142 https://github.com/typetools/checker-framework/issues/67
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Issue1142 {
+
+    void foo() {
+
+        ConcurrentHashMap<Integer, @Nullable Integer> chm = new ConcurrentHashMap<>();
+        chm.put(1, 1);
+        chm.put(3, 3);
+        chm.put(4, null);
```

Comment:
Yes I agree,
But it is unnecessary to throw error here because at line 9 there is already an error. The programmer needs to solve it first to reach to line 12.
If someone writes : 
ConcurrentHashMap<Integer, Integer> chm = new ConcurrentHashMap<>();
Then in this case line 12 will throw an error stating:
found: null
required: @Initialized @NonNull
This way it raises lesser number of warnings making easier for the programmer to debug.